WordPress/src/main/res/values/strings.xml

```diff
@@ -3951,7 +3951,7 @@ translators: %s: Select control option value e.g: "Auto, 25%". -->
     <string name="restore_progress_description_with_two_parameters">We\'re restoring your site back to %1$s %2$s.</string>
     <string name="restore_progress_icon_content_description">Restore site icon</string>
     <string name="restore_progress_footnote">No need to wait around. We\'ll notify you when your site has been restored.</string>
-    <string name="restore_progress_label">%1$s%%</string>
+    <string name="restore_progress_label">%1$s</string>
```

Comment:
Now that the `%%` is not part of the string to be localized because that's handled by the `PercentFormatter`, should this `restore_progress_label` value still be localized and go thru a `strings.xml` entry — as opposed to being used directly in the code via `UiStringText(percentFormatter.format(restoreStatus.progress ?: 0))` and dropping that `strings.xml` entry?